src/data/Thread.js

```diff
@@ -15,6 +15,8 @@ const ThreadMessage = require("./ThreadMessage");
 
 const {THREAD_MESSAGE_TYPE, THREAD_STATUS, DISCORD_MESSAGE_ACTIVITY_TYPES} = require("./constants");
 
+const escapeFormattingRegex = new RegExp("[_`~*]", "g");
```

Comment:
Have you thought about the pipe character for spoilers?

Reply:
Fixed